draft-ietf-tls-esni.md

```diff
@@ -656,9 +663,10 @@ structure available for the server, it SHOULD send a GREASE {{?RFC8701}}
   vary to exercise all supported configurations, but MAY be held constant for
   successive connections to the same server in the same session.
 
-- Set the "config_digest" field to a randomly-generated string of hash_length
-  bytes, where hash_length is the length of the hash function associated with
-  the chosen HpkeCipherSuite.
+- Set the "config_digest" field to a randomly-generated string of `Nh` bytes,
+  where `Nh` is the output length of the `Extract` function of the KDF
+  associated with the chosen cipher suite. (The KDF API is specified in
+  {{!I-D.irtf-cfrg-hpke}}.)
```

Comment:
```suggestion
  Section 4 of {{!I-D.irtf-cfrg-hpke}}.)
```

(I had some difficultly locating it without a more specific pointer.)

Reply:
Should we avoid referring to sections numbers in drafts, in case they change?

Reply:
It's easy to fix.  If HPKE was so unstable, I'd be less inclined to depend on it.